components/cluster/corosync/Makefile

```diff
@@ -15,6 +15,8 @@
 
 include ../../../make-rules/shared-macros.mk
 
+PATH= $(PATH.gnu)
+
 COMPONENT_NAME= corosync
 COMPONENT_VERSION= 2.4.2
```

Comment:
If you don't change COMPONENT_VERSION you should add (or increment if already there) COMPONENT_REVISION.